jni/include/nmslib_stream_support.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+#ifndef OPENSEARCH_KNN_JNI_NMSLIB_STREAM_SUPPORT_H
+#define OPENSEARCH_KNN_JNI_NMSLIB_STREAM_SUPPORT_H
+
+#include "jni_util.h"
+#include "native_engines_stream_support.h"
+
+#include <jni.h>
+#include <stdexcept>
+#include <iostream>
+#include <cstring>
+
+namespace knn_jni {
+namespace stream {
+
+
+
+/**
+ * TODO : KDY
```

Comment:
What is KDY?

Reply:
oh shoot, that's the placeholder for comment. 😅 
and I will update it.